### PR TITLE
feat(lmendev): undefined academy readme.md

### PR DESCRIPTION
**Pull request by Lmendev**
## Context
First week activity

1. Fork https://github.com/undefined-academy/week-11
2. Clone forked repository to HOME folder.
3. Add personal GitHub-profile markdown file inside `github-profiles` folder. Filename should be Discord user including the numeric tag separated by `-`.
4. Include personal GitHub-profile link.
5. Commit it using git.
6. Create Pull Request to https://github.com/undefined-academy/week-11

Pull Request description should not be empty.

## In this PR
Pull request created according to required instructions

### CODE REVIEW

**Review by Cristhian-Medina — APPROVED** [review]

File: github-profiles/lmendev-2708/lmendev-2708.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+https://github.com/Lmendev
```
Comment (Psychobroder) [inline github-profiles/lmendev-2708/lmendev-2708.md]: [lmendev](https://github.com/lmendev)

a wise man once told me, you should change this to make a link to your profile ;)

File: github-profiles/lmendev-2708/lmendev-2708.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+<img align='right' width='30%' src='https://media3.giphy.com/media/iIqmM5tTjmpOB9mpbn/giphy.gif' alt='developer gif' />
+
+* 🌱 Now, I am learning about react and TypeScript
+* ✍🏼 I'm writing about <a href='http://procesadormipslogisim.blogspot.com/'>simulating a MIPS processor</a> on my blog, check it out!
+* 🎹 Aside from coding I enjoy play piano
+* ✉️ <a href="mailto:lmendoza.mg@gmail.com?subject=I%20came%20across%20your%20GitHub%20profile!">Send me an email</a>, asking for help or just to say hi! 🙋🏽‍♂️
+
+## Last learned commands
+
+| Command | Description |
```
Comment (Psychobroder) [inline github-profiles/lmendev-2708/lmendev-2708.md:20]: in the tables you could give a format for a better comprehension

**Review by Psychobroder — APPROVED** [review]
great work! with that changes to me looks enough to get ready for the next week!

File: github-profiles/lmendev-2708/lmendev-2708.md (hunk 1 above)
Comment (Lmendev) [reply github-profiles/lmendev-2708/lmendev-2708.md]: Oh!! 😢 this is outdated, it was corrected in https://github.com/undefined-academy/semana-1/pull/473/commits/076c8689bdaebec604f1f68fc4f14c22e70c02cf 8 hours ago :)

**Comment by Lmendev** [comment]
Estos cambios fueros incluidos en https://github.com/undefined-academy/semana-1/pull/494

**Pull request closed without merging**